packages/remix-dev/config/defaults/entry.server.spa.tsx

```diff
@@ -1,20 +1,73 @@
-import type { EntryContext } from "@react-router/node";
+import { PassThrough } from "node:stream";
```

Comment:
I think we can just remove this file entirely and the condition that uses it in the remix dev and remix reveal code?

https://github.com/remix-run/react-router/blob/v7/packages/remix-dev/config.ts#L493
https://github.com/remix-run/react-router/blob/v7/packages/remix-dev/cli/commands.ts#L148

Reply:
Ah I see what you did - you got rid of the `prohibitOutOfOrderStreaming` check and just made the spa version always use `handleBotRequest`.  IMO, it would be fine to just delete the SPA version and let `npx remix reveal` expose the default node one with the `prohibitOutOfOrderStreaming` check

Reply:
Took a stab at that in [dd6a86c](https://github.com/remix-run/react-router/pull/11522/commits/dd6a86c2216bc3b082016212bc0cc0b2980e2143) - feel free to revert if we want to keep it 👍

Reply:
hm - so that broke  a handful of e2e tests so I reverted for now :)